src/raft/raft.h

```diff
@@ -100,6 +100,8 @@ namespace raft
 
     static constexpr size_t append_entries_size_limit = 20000;
     size_t entry_size_not_limited = 0;
+    size_t tx_count = 0;
```

Comment:
For the sake of consistency, let's try to stick with the raft terminology. Raft deals with entries, not transactions.